### ISSUE TITLE
Accessibility | Implement a skip link

### ISSUE THREAD

**Issue by f4-lex** [Frachtwerk/essencium-frontend#838]
**Description**
Right now, a Keyboard user has to tab through the Top-Bar, Navigation-Drawer, and Footer every time they load a page. A keyboard only [skip link](https://www.barrierefreies-webdesign.de/knowhow/sprunglinks/) should be implemented to allow them to skip directly to the main page.

- [ ] implement changes
- [ ] update docs
